src/main/java/edu/cornell/kfs/fp/dataaccess/impl/RecurringDisbursementVoucherSearchDaoJdbc.java

```diff
@@ -0,0 +1,96 @@
+package edu.cornell.kfs.fp.dataaccess.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.rice.core.framework.persistence.jdbc.dao.PlatformAwareDaoBaseJdbc;
+import org.kuali.rice.kew.api.KewApiConstants;
+import org.kuali.rice.krad.service.DataDictionaryService;
+import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
+import org.kuali.kfs.sys.context.SpringContext;
+
+import org.springframework.jdbc.core.ConnectionCallback;
+
+import edu.cornell.kfs.fp.dataaccess.RecurringDisbursementVoucherSearchDao;
+
+public class RecurringDisbursementVoucherSearchDaoJdbc extends PlatformAwareDaoBaseJdbc implements RecurringDisbursementVoucherSearchDao {
+
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(RecurringDisbursementVoucherSearchDaoJdbc.class);
+
+    protected DateTimeService dateTimeService;
+
+    @Override
+    public Collection<String> findSavedDvIdsSpawnedByRecurringDvForCurrentAndPastFiscalPeriods(Date currentFisalPeriodEndDate) {
+        final Date currentFisalPeriodEndDateForSql= currentFisalPeriodEndDate;
+        Collection<DisbursementVoucherDocument> dvsFound = null;
+
+        return getJdbcTemplate().execute(new ConnectionCallback<Collection<String>>() {
+            public Collection<String> doInConnection(Connection con) throws SQLException {
+                PreparedStatement selectStatement = null;
+                ResultSet queryResultSet = null;
+                List<String> finalResults = new ArrayList<String>();
+
+                try {
+                    String selectStatementSql = getSavedDvsSpawnedByRecurringDvForCurrentAndPastFiscalPeriodsSelectSql(currentFisalPeriodEndDateForSql);
+                    LOG.info("SQL Statement to obtain saved DVs spawned by RCDV for current and past fiscal period follows.");
+                    LOG.info("*************************************");
+                    LOG.info(selectStatementSql);
+                    LOG.info("*************************************");
+                    selectStatement = con.prepareStatement(selectStatementSql);
+                    queryResultSet = selectStatement.executeQuery();
+
+                    while (queryResultSet.next()) {
+                        finalResults.add(queryResultSet.getString(1));
```

Comment:
I would get the queryResultSet.getString(1) and queryResultSet.getString(2) into local variables and then use those.